src/components/Map/Map.test.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {
+  act,
+  fireEvent,
+  render as renderRtl,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { mockMediaQuery } from '@test/testUtils';
+
+import type { MapProps } from './Map';
+import { Map } from './Map';
+
+const mediaQueryBreakPoint = 500;
+const { setScreenWidth } = mockMediaQuery(mediaQueryBreakPoint);
+
+const user = userEvent.setup();
+
+window.scrollTo = jest.fn();
+
+describe('Map', () => {
+  beforeEach(() => {
+    setScreenWidth(mediaQueryBreakPoint + 100);
+  });
+
+  describe('Marker', () => {
+    it('should show marker when marker is set', () => {
+      render({ marker: [59.2641592, 10.4036248] });
+      expect(locationMarker()).toBeInTheDocument();
+    });
+
+    it('should not show marker when marker is not set', () => {
+      render({ marker: undefined });
+      expect(locationMarker()).not.toBeInTheDocument();
+    });
+  });
+
+  describe('Click map', () => {
+    it('should call mapClicked with correct coordinates when map is clicked', async () => {
+      const handleMapClicked = jest.fn();
+      render({
+        mapClicked: handleMapClicked,
+      });
+
+      clickMap();
+
+      expect(handleMapClicked).toHaveBeenCalledWith(
+        59.265880628258095,
+        10.371093750000002,
+      );
+    });
+
+    it('should get different coordinates when map is clicked at a different location', () => {
+      const mapClicked = jest.fn();
+      render({
+        mapClicked: mapClicked,
+      });
+
+      clickMap(50, 50);
```

Comment:
What is the relation between the args in clickMap and the expected result?

Reply:
Can the view size be locked somehow so the coordinates will stay a bit more consistent?
I'm worried this will become an issue and if this fails on a different system than yours. There really isn't much to go on in terms of fixing it.
Is it necessary to include such a test to get the enough coverage?
Maybe this test is more suited as an integration. Or instead of comparing the coordinates, maybe you could be better off verifying if the marker is on the map as a result of the click?

Reply:
I can rewrite the test to check that a map click will change the coordinates, but not check on the exact coordinates returned. I think this will be a good enough test. And it will not fail on other systems.